File: Practica-java/src/_2_Expresiones/Ejercicio_1.java
package _2_Expresiones;
import java.util.Scanner;
/* 1. Escribe la sigiente expresion como expresion en C++:
*  a) a/b + 1
*/
public class Ejercicio_1 {
    public static void main(String[] arg) {
        float a,b,resultado;
        Scanner entrada = new Scanner(System.in);
        System.out.println( "Ingrese el valor de a: ");
        a=entrada.nextFloat();
        System.out.println( "Ingrese el valor de b: ");
        b=entrada.nextFloat();

        resultado = a/b + 1;
        System.out.println("Resultado: "+resultado);
    }
}

File: Practica-cpp/Trabajo_practicos/2-Expresiones/Ejercicio_1.cpp
/*
* 1. Escribe la sigiente expresion como expresion en C++:
 *  a) a/b + 1
 */
#include <iostream>
using namespace std;

int main() {
 float a,b,resultado;
 cout<<"Ingrese el valor de a: ";cin>>a;
 cout<<"Ingrese el valor de b: ";cin>>b;

 resultado = a/b + 1;
 cout.precision(2);
 cout<<"Resultado: "<<resultado;

 return 0;;
}

/*
 * 2. Escribe la sigiente expresion como expresion en C++:
 *  b) (a+b)/(c+d)
 */
